config/locales/doc_auth/en.yml

```
@@ -193,11 +193,12 @@ en:
       switch_back_image: Arrow pointing from phone to computer
       test_ssn: In the test environment only SSNs that begin with “900-” or “666-” are
         considered valid. Do not enter real PII in this field.
+      text1: Your ID cannot be expired.
       text2: You will not need the card with you.
-      text3: You do not need to be the primary account holder on your phone plan. If
-        your phone number doesn’t work, we can send you a verification code by
-        mail.
-      welcome: 'To verify your identity, you will need:'
+      text3_html: We’ll call or text this number to verify your identity. If we can’t
+        verify your phone number, you can verify by mail instead.
+        %{learn_more_link}
```

if the `%{learn_more_link}` is at the end of this string, we probably don't need it inside the string at all, right? We can just add that in the template, right?
```suggestion
      text3: We’ll call or text this number to verify your identity. If we can’t
        verify your phone number, you can verify by mail instead.
```

and then in the template:

```erb
        <p>
          <%= t('doc_auth.instructions.text3') %>
          <%= new_window_link_to(
                  t('idv.troubleshooting.options.learn_more_address_verification_options'),
                  help_center_redirect_path(
                    category: 'verify-your-identity',
                    article: 'phone-number-and-phone-plan-in-your-name',
                    flow: :idv,
                    step: :welcome,
                    location: 'missing_items',
                  ),
              ) %>
        </p>
```